app/Http/Kernel.php

```diff
@@ -16,7 +16,6 @@ class Kernel extends HttpKernel
     protected $middleware = [
         // \App\Http\Middleware\TrustHosts::class,
         \App\Http\Middleware\TrustProxies::class,
-        \Fruitcake\Cors\HandleCors::class,
```

Comment:
removed in v10?

Reply:
I forgot to replace with Laravel default CORS middle and I will include in next commit.